Update sspi-status-codes.md (#1694)
MicrosoftDocs/win32: desktop-src/SecAuthN/sspi-status-codes.md

| SEC\_E\_CRYPTO\_SYSTEM\_INVALID<br/>         | The cryptographic system or checksum function is not valid because a required function is unavailable.<br/>                                                                                                                                                                                                                                 |
| SEC\_E\_DECRYPT\_FAILURE<br/>                | The specified data could not be decrypted.<br/>                                                                                                                                                                                                                                                                                             |
| SEC\_E\_DELEGATION\_REQUIRED<br/>            | The requested operation cannot be completed. The computer must be trusted for delegation, and the current user account must be configured to allow delegation.<br/>                                                                                                                                                                         |

> | SEC\_E\_DOWNGRADE\_DETECTED<br/>             | The system detected a possible attempt to compromise security. Verify that the server that authenticated you can be contacted.<br/>NOTE: This error can be generated simply due to lack of connectivity to a domain controller, and may not be indicative of malicious activity at all.                                                                                                                                                                                                         |

Reviewer: ```suggestion
| SEC\_E\_DOWNGRADE\_DETECTED<br/>             | The system detected a possible attempt to compromise security. Verify that the server that authenticated you can be contacted.<br/><b>NOTE:</b> This error can be generated simply due to lack of connectivity to a domain controller and may not be indicative of malicious activity.                                                                                                                                                                                                         |
```
